website/docs/routes/university.md

```diff
@@ -587,6 +587,18 @@ Note：[Source website](https://ece.umass.edu/seminar) may be empty when there's
 
 </Route>
 
+## 成都锦城学院 {#cheng-du-jin-cheng-xue-yuan}
+
+### 教务处 {#cheng-du-jin-cheng-xue-yuan-jiao-wu-chu}
+
+<Route author="nyaShine" example="/scujcc/jwb/tzgg" path="/scujcc/jwb/:category?" paramsDesc={['分类ID']} radar="1">
```

Comment:
Please mention the default value of `category` if it's optional.
https://docs.rsshub.app/joinus/new-rss/add-docs